src/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -116,6 +116,9 @@ public void createDataSource() throws SQLException {
         int numConnections =
             (workConf.getNumDBConnections() + workConf.getNodes().size() - 1) / workConf.getNodes().size();
         for (String ip : workConf.getNodes()) {
+            if (listDataSource.size() != 0) {
```

Comment:
Comment on why we do this. Also, lets parametrize this 5000 to the xml conf.

Reply:
Done